include/rx/ranges.hpp

```diff
@@ -2317,6 +2317,109 @@ template <class... Inputs>
     return ZipLongestRange(std::forward_as_tuple(as_input_range(std::forward<Inputs>(inputs))...));
 }
 
+/// Forward all elements of the range as-is.
+struct passthrough {
```

Comment:
Why is this useful?

Reply:
It's meant for cases when you cannot not have a value. Default return types and such.

Reply:
I'm not sure I understand. Can you elaborate with an example?